tsp_simple.cc

```diff
@@ -135,8 +137,16 @@ void MissionsBuilder(const TSPTWDataDT& data, RoutingModel& routing,
         }
         if (due.back() < CUSTOM_MAX_INT) {
           if (late_multiplier > 0) {
-            routing.GetMutableDimension(kTime)->SetCumulVarSoftUpperBound(
-                index, due.back(), late_multiplier);
+            if (initial_value >= 0) {
+              assignment->Add(cumul_var);
+              if (initial_value > due.back())
+                assignment->SetMax(cumul_var, initial_value);
+              else
+                assignment->SetMax(cumul_var, due.back());
```

Comment:
Why don't we do this creation inside the function routeBuilder/initialSolutionBuilder, instead of moving `assignment` inside each function? Before this modification, the functions were separated by functionality (creation of services, vehicles, initial solution etc.) but now the logic of initial solution creation leaks to other functions. What  do you think?

Reply:
In fact just realized, here we can do `cumul_var->SetMax(due.back());` or `cumul_var->SetMax(initial_value);` as well (to cut the search space a bit)  then we have to keep the logic here :/ I  don't know what's the best.. If you want, you think about the ideas here and make a suggestion.

Reply:
Ah no this is not possible (in general) :/ It work for pushing the pause but it wouldn't work for general, in case we can decrease the total cost by increasing the lateness of a single job. So we can't do `cumul_var->SetMax(initial_value);` in general.

Reply:
Do we want to push the break or to re-optimize the route individually ?

Reply:
Both I am afraid. However, the general case should be able to handle (or it should be enough) for the purpose of moving the pause, no?

Reply:
@braktar on my end, neither `setMax` nor `setValue` work... when I launch the optimization with the result of the previous one (with `setValue` the objective function calculated at iteration 0 is still incorrect (as if I didn't set the value of each variable)... :thinking: